package.json

```diff
@@ -24,48 +24,47 @@
     "node": ">=6.9"
   },
   "scripts": {
-    "start": "npm run init:generated && npm run fetch && node ./antwar.bootstrap.js develop",
-    "start-only": "node ./antwar.bootstrap.js develop",
-    "build": "npm run init:generated && npm run fetch && rm -rf build/ && node ./antwar.bootstrap.js build && npm run sitemap && echo webpack.js.org > build/CNAME",
-    "build-test": "npm run build && http-server build/",
-    "deploy": "gh-pages -d build",
+    "start": "webpack-dev-server --config webpack.dev.js --env.dev",
+    "build": "webpack --config webpack.prod.js",
     "fetch": "sh src/scripts/fetch.sh",
-    "init:generated": "mkdirp ./generated/loaders && mkdirp ./generated/plugins ",
+    "test": "npm run lint",
```

Comment:
I would like to have e2e tests, using puppeteer or cypress.

Reply:
Though definitely out of scope here, sounds like an overkill for linting `*.md` files, in which situation you think e2e tests will cover something we wouldn't be able to with this setup?

Reply:
Just to verify that the page works, all pages are there, Gitter chat works, mobile sidebar works, Search too, etc.